Libraries/rocSPARSE/level_2/bsrmv/README.md

```diff
@@ -13,8 +13,8 @@ where
 - $A'$ is a sparse matrix in BSR format with `rocsparse_operation` and described below.
 
 ## Application flow
-1. Setup a sparse matrix in BSR format. Allocate an x and a y vector and set up $\alpha$ and $\beta$ scalars.
-2. Setup a handle, a matrix descriptor and a matrix info.
```

Comment:
It is better to separate feature and formatting commits. There is no need to make already big PRs even bigger.

Reply:
So then I can drop the [Fix and clarify rocsparse examples device synchronization](https://github.com/amd/rocm-examples/pull/90/commits/d43ffd6bbdf88e25119388753b9bf5d82f7dac4d) commit, I think it's the one introducing this many formatting/comment fixes